components/NavDrawer/NavDrawer.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import {
+  Drawer,
+  DrawerContent,
+  DrawerDescription,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerTitle,
+  DrawerTrigger,
+} from './NavDrawer';
+
+Object.defineProperty(window, 'matchMedia', {
+  writable: true,
+  value: jest.fn().mockImplementation((query) => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(),
+    removeListener: jest.fn(),
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
+    dispatchEvent: jest.fn(),
+  })),
+});
+
+describe('Drawer Component', () => {
+  it('Drawer opens and closes properly', async () => {
+    render(
+      <Drawer>
```

Comment:
The `shouldScaleBackground` prop is set to true by default. However, from what I read, the element also needs to have the `[vaul-drawer-wrapper]` data attribute to work correctly. It looks like we might have missed that for now, so no matter what value we pass, the background won’t scale. I'm not sure if that's something that should be fixed or if we should just leave it be?

Reply:
My opinion is that it should probably be changed if any reference to it is being introduced/included but if you think it would be better to have a larger conversation with Shashi, for example, I'm game.

Reply:
> The `shouldScaleBackground` prop is set to true by default. However, from what I read, the element also needs to have the `[vaul-drawer-wrapper]` data attribute to work correctly. It looks like we might have missed that for now, so no matter what value we pass, the background won’t scale. I'm not sure if that's something that should be fixed or if we should just leave it be?

Just leave it as is. Unless it's an issue with your ticket, let's create a ticket for it in the backlog and move on.